.github/workflows/deploy-zeit.yml

```diff
@@ -30,6 +30,7 @@ jobs:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           zeit-token: ${{ secrets.ZEIT_TOKEN }}
           zeit-team-id: team_zpOFzrZjyURkFwGZsKEEHzcv
+          now-args: '--confirm'
```

Comment:
https://zeit.co/docs/now-cli#commands/now/unique-options/confirm
to avoid the questions on CLI during the CI :)